### DISCUSSION TITLE
Existing session default to saving session with its name

### DISCUSSION THREAD

**Discussion by rieje** [jarun/nnn#1942] [General]
Shouldn't an existing session default to saving session with its name? I have tmux with multiple instances of nnn sessions but I don't see a way to know which session is which either.

**Comment by jarun** [comment]
Please share the steps and where you see a problem.

**Reply by rieje** [reply]
`nnn -Ss sess-1`, `s` to show session options, `s` again to save session, prompted for session name.

**Comment by jarun** [comment]
Implemented at commit 00fb7b6f6b6cf8e4b0936ba972a0943f7da4382b.

**Comment by jarun** [comment]
Session name is shown in help at commit 1a849f1847a989b13c6b8623b128734689323128.
